doc/project-plan/communication-plan.rst

```diff
@@ -0,0 +1,76 @@
+=====================================
+General Communication Management Plan
+=====================================
+
+Summary
+~~~~~~~
+
+Investigate the beginnings of an opinionated
+new storage appliance project in an collaborative fashion. See the
+:ref:`aquarium-project-plan` for more information.
+
+Communication Goals
+~~~~~~~~~~~~~~~~~~~
+
+- Share information about on-going development and project achievements globally
+- Keep stakeholders informed of project timeline and project needs
+- Provide clear insight into any decisions made or ongoing discussions
+- Engage community in feedback driven discussions and decision making
+
+
+.. list-table::
+   :widths: 15 45 10 10 10 10
+   :header-rows: 1
+
+   * - Communication Type
+     - Description
+     - Frequency
+     - Format
+     - Distribution
+     - Owner
+   * - Weekly check-ins
+     -
+         - Weekly meeting (x3 per timezone) to discuss Aquarium.
+         - Meeting minutes that are regularly sent to the mailing list.
```

Comment:
What mailing list are we going to use for these minutes?

Reply:
So, I've left this intentionally blank. Currently, we're sending to ceph@suse.de but we really should have our own public ML. @l-mb is graciously looking into this.

Reply:
Consider this a "to be updated" thing. We need to go through SUSE infra for a ML. I've added an issue for myself here: https://github.com/aquarist-labs/aquarium/issues/199

Reply:
Sounds good. I figured you were already working on a solution here. ;)